corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -635,6 +635,9 @@ FormplayerFrontend.reqres.setHandler('clearUserData', function () {
 });
 
 FormplayerFrontend.on('navigateHome', function () {
+    // switches tab back from the application name
+    document.title = "Web Apps - CommCare HQ";
```

Comment:
Can you wrap the string in a call to `gettext`? That'll make sure it gets translated.